### PR TITLE
Fix typos in 3 DC in 2 cities deployment doc

### PR DESCRIPTION
**Pull request by TomShawn**
### What is changed, added or deleted? (Required)

Fix typos in 3 DC in 2 cities deployment doc.

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### CODE REVIEW

File: three-data-centers-in-two-cities-deployment.md [hunk 1]
```diff
@@ -162,7 +162,7 @@ tikv_servers:
 
 在两地三中心的架构部署中，从性能优化的角度，除了常规参数配置外，还需要对集群中相关组件参数进行调整。
 
-- 启用 tikv grpc消息压缩 由于涉及到集群中的数据在网络中传输，需要开启 gRPC 消息压缩，降低网络流量。
+- 启用 TiKV gRPC 消息压缩 由于涉及到集群中的数据在网络中传输，需要开启 gRPC 消息压缩，降低网络流量。
```
Comment (lilin90) [inline three-data-centers-in-two-cities-deployment.md:165]:
```suggestion
- 启用 TiKV gRPC 消息压缩。由于涉及到集群中的数据在网络中传输，需要开启 gRPC 消息压缩，降低网络流量。

```

File: three-data-centers-in-two-cities-deployment.md [hunk 2]
```diff
@@ -181,13 +181,13 @@ tikv_servers:
     raftstore.raft-max-election-timeout-ticks: 1200
     ```
 
-- 调度设置，在集群启动后，通过 `tiup ctl pd` 工具进行调度策略修改。修改 TiKV raft 副本数按照安装时规划好的副本数进行设置，在本例中为 5 副本。
+- 调度设置，在集群启动后，通过 `tiup ctl pd` 工具进行调度策略修改。修改 TiKV Raft 副本数按照安装时规划好的副本数进行设置，在本例中为 5 副本。
```
Comment (lilin90) [inline three-data-centers-in-two-cities-deployment.md:184]:
```suggestion
- 调度设置。在集群启动后，通过 `tiup ctl pd` 工具进行调度策略修改。修改 TiKV Raft 副本数按照安装时规划好的副本数进行设置，在本例中为 5 副本。

```

File: three-data-centers-in-two-cities-deployment.md [hunk 3]
```diff
@@ -18,7 +18,7 @@ TiDB 分布式数据库通过 Raft 算法原生支持两地三中心架构的建
 
 本文以北京和西安为例，阐述 TiDB 分布式数据库两地三中心架构的部署模型。
 
-例中，北京有两个机房 IDC1 和 IDC2，异地西安一个机房 IDC3。北京同城两机房之间网络延迟低于 3ms，北京与西安之间的网络使用 ISP 专线，延迟约 20ms。
+例中，北京有两个机房 IDC1 和 IDC2，异地西安一个机房 IDC3。北京同城两机房之间网络延迟低于 3 ms，北京与西安之间的网络使用 ISP 专线，延迟约 20 ms。
```
Comment (lilin90) [inline three-data-centers-in-two-cities-deployment.md:21]:
```suggestion
本例中，北京有两个机房 IDC1 和 IDC2，异地西安一个机房 IDC3。北京同城两机房之间网络延迟低于 3 ms，北京与西安之间的网络使用 ISP 专线，延迟约 20 ms。

```

**Comment by lilin90** [comment]
Would you please also fix descriptions in [操作系统性能参数调优](https://docs.pingcap.com/zh/tidb/dev/tune-operating-system#%E5%86%85%E5%AD%98%E9%80%8F%E6%98%8E%E5%A4%A7%E9%A1%B5)?

**Comment by TomShawn** [comment]
> Would you please also fix descriptions in [操作系统性能参数调优](https://docs.pingcap.com/zh/tidb/dev/tune-operating-system#%E5%86%85%E5%AD%98%E9%80%8F%E6%98%8E%E5%A4%A7%E9%A1%B5)?

Done in https://github.com/pingcap/docs-cn/pull/3829/commits/1e36b7f85851b2f34961b0a70f47680393cb8de5.

**Pull request merged**